libs/ledger-live-common/src/families/polkadot/logic.ts

```diff
@@ -1,2 +1,2 @@
 // Encapsulate for LLD et LLM
-export * from "@ledgerhq/coin-polkadot/logic";
+export * from "@ledgerhq/coin-polkadot/utils";
```

Comment:
I have trouble understanding this change with just the PR

Reply:
understood, i just find it weird to have a logic.ts file with an export of utils but indeed it makes sense